CODE_REVIEW.md

```diff
@@ -0,0 +1,23 @@
+2) 
```

Comment:
Code improvements should not be part of task 3. It should be part of task1

Reply:
Done